LICENSE.txt

```diff
@@ -156,34 +156,6 @@ The above copyright notice and this permission notice shall be included in all c
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-IronPython.StdLib v.2.7.9:
```

Comment:
this is used by our iron pythion package 
Keep it or no ?

Reply:
I think either keep it here or we need to consider adding about box content for common packages?

Reply:
@Amoursol @Jingyi-Wen something to consider for the future - how should our packages make about box info available?

Reply:
@mjkkirschner is the desire to show About Box licence info for packages specifically for IronPython? Or generally? Either way, we should do it.

Reply:
I am putting this back